xsd-core/src/main/java/io/ballerina/xsd/core/Response.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.xsd.core;
+
+import io.ballerina.xsd.core.diagnostic.XsdDiagnostic;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Response {
+    private final String types;
+    private List<XsdDiagnostic> diagnostics = new ArrayList<>();
+
+    public Response(String types, List<XsdDiagnostic> diagnostics) {
+        this.types = types;
+        this.diagnostics = diagnostics;
+    }
+
+    public String getTypes() {
+        return types;
+    }
+
+    public List<XsdDiagnostic> getDiagnostics() {
+        return diagnostics;
+    }
+
+    public void setDiagnostics(List<XsdDiagnostic> diagnostics) {
+        this.diagnostics = diagnostics;
+    }
+}
```

Comment:
Cant we use a record for this?

Reply:
Fixed in [700f1e6](https://github.com/ballerina-platform/xsd-tools/pull/1/commits/700f1e6eab35e1aa5219b25114b11c11875b5472)